staging/src/k8s.io/apiserver/pkg/util/peerproxy/peerproxy_handler.go

```diff
@@ -93,12 +89,12 @@ func (h *peerProxyHandler) HasFinishedSync() bool {
 }
 
 func (h *peerProxyHandler) WaitForCacheSync(stopCh <-chan struct{}) error {
-
-	ok := cache.WaitForNamedCacheSync("unknown-version-proxy", stopCh, h.storageversionInformer.HasSynced, h.storageversionManager.Completed)
+	go h.apiserverIdentityInformer.Run(stopCh)
```

Comment:
Seems weird to run the informer here. Should run it in `pkg/controlplane/apiserver/server.go` where this function is called